sdk/identity/azure-identity/tests/integration/azure-kubernetes-service/app.py

```diff
@@ -0,0 +1,50 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+import os
+import asyncio
+
+from azure.identity import ManagedIdentityCredential
+from azure.identity.aio import ManagedIdentityCredential as AsyncManagedIdentityCredential
+from azure.storage.blob import BlobServiceClient
+from azure.storage.blob.aio import BlobServiceClient as AsyncBlobServiceClient
+
+
+def run_sync():
+    credential = ManagedIdentityCredential()
+
+    client = BlobServiceClient(
+        account_url=f"https://{os.environ['IDENTITY_STORAGE_NAME']}.blob.core.windows.net",
+        credential=credential,
+    )
+
+    containers = client.list_containers()
+    for container in containers:
+        print(container["name"])
+
+    print(f"Successfully acquired token with ManagedIdentityCredential")
+
```

Comment:
Why we don't need to close client/cred here?

Reply:
Found that it wasn't explicitly necessary with sync client/credentials. I added that with the async credentials to prevent output like "Unclosed client session" which are prolific if not closing or using async context managers.